tox.ini

```diff
@@ -8,6 +8,7 @@ setenv =
     PYTHONPATH = {toxinidir}
 commands =
     py.test --cov --cov-report= {posargs}
+whitelist_externals = make
```

Comment:
Why did you add this?

Reply:
Without it `tox` throws a warning:
`pylint runtests: commands[0] | make pylint
WARNING: test command found but not installed in testenv
  cmd: /usr/bin/make
  env: /Users/meti/Documents/Dev/xknx/.tox/pylint
Maybe you forgot to specify a dependency? See also the whitelist_externals envconfig setting.`
on my system. Same for 
`pydocstyle runtests: commands[0] | make pydocstyle`

Reply:
Could this be the reason the coverage decreased? These warnings occurred with the tests of the last PR #161 in travis too. 
See for example https://travis-ci.org/XKNX/xknx/jobs/471596429 line 589 ff.